### PR TITLE
add stubed help request and sample env

### PR DESCRIPTION
**Pull request by maysakanoni**
**What**
- Add a stubbed help request to test if the lambda can query the `cv-19-res-support` api 

**Why**
- The lambda has the same subnets as the api, but we want to ensure it is able to create a help request as anticipated

**Notes**
- Once this is tested this pr will be reverted

### CODE REVIEW

File: SAMPLE.env [hunk 1]
```diff
@@ -0,0 +1 @@
+CV-19-RES-SUPPORT-V3-HELP-REQUESTS-URL=
```
Comment (kosiakkatrina) [inline SAMPLE.env:1]: new line? 😄

File: lib/gateways/here_to_help_api.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ class HereToHelpGateway:
 
     def create_help_request(self, help_request):
         try:
-            help_requests_url = os.getenv("CV-19-RES-SUPPORT-V3-HELP-REQUESTS-URL")
+            help_requests_url = os.getenv("CV_19_RES_SUPPORT_V3_HELP_REQUESTS_URL")
```
Comment (kosiakkatrina) [inline lib/gateways/here_to_help_api.py:15]: this is different from sample.env?

File: lib/lambda_handler.py [hunk 3]
```diff
@@ -3,7 +3,67 @@ def __init__(self, create_help_request):
         self.create_help_request = create_help_request
 
     def execute(self, event, context):
-        help_request = [{}]
+        help_request = [
+            {
+                "Id": 1,
```
Comment (kosiakkatrina) [inline lib/lambda_handler.py:8]: Hm, probably don't want an id here. Api should generate a new one

Comment (maysakanoni) [reply lib/lambda_handler.py:8]: yess

File: lib/gateways/here_to_help_api.py (hunk 2 above)
Comment (maysakanoni) [reply lib/gateways/here_to_help_api.py:15]: forgot to update sample env!

**Comment by maysakanoni** [comment]
testing how dependencies are packaged using circle

**Pull request closed without merging**